migrations.go

```diff
@@ -53,7 +53,7 @@ func migrate_4_SwarmKey(cfg *Config) bool {
 // for bootstrap nodes.
 // Replaces all bootstrap nodes with default values if so.
 func migrate_5_Bootstrap_node(cfg *Config) bool {
-	obsoleteBootstrapNodeList := []string{"3.120.224.94"}
+	obsoleteBootstrapNodeList := []string{"3.120.224.94", "52.57.56.230"}
```

Comment:
i think we need a separate function for this since we are detecting obsolete testnet  bootstrap nodes not the prod bootstrap nodes.  this funct currently only replaces the nodes in the regular bootstrap section.

Reply:
ok, i see.

Reply:
it can be done in the same func, just need a small flag.

Reply:
anyway, let you guys decide